components/cloud-event-gateway-proxy/pkg/handler/handler.go

```diff
@@ -21,17 +21,21 @@ import (
 )
 
 const (
-	// noDuration signals that the dispatch step hasn't started
+	// noDuration signals that the dispatch step has not started yet.
 	noDuration = -1
 )
 
 var (
+	// additionalHeaders the required headers by EMS for publish requests.
```

Comment:
I think it reads better like this: What do you think? 

```suggestion
	// additionalHeaders are the required headers by EMS for publish requests.
```

Reply:
Done.